datahub-web-react/src/graphql/chart.graphql

```diff
@@ -110,6 +110,9 @@ query getChart($urn: String!) {
                 }
             }
         }
+        genericDownstreamLineage {
```

Comment:
So in this case, it really isn't "generic" in the sense that only Charts can appear as downstreams.. (similar to UpstreamLineage of Datasets)

Reply:
Is the idea to make UpstreamLineage also generic?

Reply:
Exactly- we'll also want to do this for Upstream